src/styles/_variables.scss

```diff
@@ -9,4 +9,4 @@ $dark-olive: #beb66d;
 $teal: #009875;
 
 // Fonts
-$font-family-sans-serif: 'Lato', -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, 'Helvetica Neue', Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol'
\ No newline at end of file
+$font-family-sans-serif: 'Lato', -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, 'Helvetica Neue', Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol'
```

Comment:
I think this is just forcing a new line at the end of file. I don't think there are any changes.